### PR TITLE
web: fix react ui-kit import

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are fixing components import in selectAccount page.

### CODE REVIEW

File: web-frontend/src/pages/SelectAccount/SelectAccount.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import { toMAS } from '@massalabs/massa-web3';
 import { Link, useNavigate } from 'react-router-dom';
 import { goToErrorPage, routeFor } from '../../utils';
 import useResource from '../../custom/api/useResource';
 import { AccountObject } from '../../models/AccountModel';
+import { toMAS } from '@massalabs/massa-web3';
+
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+import { Button, AccountSelector } from '@massalabs/react-ui-kit';
+import { MassaToken } from '@massalabs/react-ui-kit/src/components/Icons/Svg/Massa/MassaToken';
```
Comment (peterjah) [inline web-frontend/src/pages/SelectAccount/SelectAccount.tsx:10]: MassaToken is not exported in the root index?
`import { Button, AccountSelector, MassaToken } from '@massalabs/react-ui-kit';` would be nice

Comment (maxime-carabina) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:10]: No! He is not export in the `index.ts`.
But I can add it in.

Comment (maxime-carabina) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:10]: > MassaToken is not exported in the root index? `import { Button, AccountSelector, MassaToken } from '@massalabs/react-ui-kit';` would be nice


No! He is not export in the index.ts.
But I can add it in.

Comment (mazmassa) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:10]: Is there any reason to use the whole path!? We should be able to import it only with from '@massalabs/react-ui-kit'

Comment (mazmassa) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx:10]: Are we exporting the SVG's on ui-kit!?

Comment (maxime-carabina) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx]: No! We doesn't export SVG on UI-KIT. I will make a PR to solve this problem.

Comment (maxime-carabina) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx]: @mazmassa Done one the ui-kit. So I will update package for kit when ui-kit is ready.

Comment (mazmassa) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx]: Great! Thanks!

Comment (maxime-carabina) [reply web-frontend/src/pages/SelectAccount/SelectAccount.tsx]: Done on branch @peterjah

**Review by peterjah — APPROVED** [review]

**Review by mazmassa — APPROVED** [review]

**Pull request merged**